src/components/PurchaseHistory/CardContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "@emotion/styled";
+import { PurChaseHistoryCard } from "./Card";
+
+export const PurchaseHistoryCont = () => {
+  return (
+    <Cont>
+      <PurchaseHistoryTitle>구매내역</PurchaseHistoryTitle>
+      <PurchaseHistoryContent>
+        <PurchaseHistoryCategory>
+          <OrderDate>주문일자</OrderDate>
+          <Quantity>수량</Quantity>
+          <Price>결제금액</Price>
+          <ShipmentState>상태</ShipmentState>
+        </PurchaseHistoryCategory>
+        <PurChaseHistoryCard />
+        <PurChaseHistoryCard />
+      </PurchaseHistoryContent>
+    </Cont>
+  );
+};
+
+const Cont = styled.section``;
+
+const PurchaseHistoryContent = styled.div`
+  margin-top: 28px;
+`;
+
+const PurchaseHistoryTitle = styled.h1`
+  font-size: 2.4rem;
+  font-weight: 700;
+`;
+
+const PurchaseHistoryCategory = styled.div`
+  display: flex;
+  padding: 0px 52px;
+  width: 100%;
+  border-bottom: 1px solid #a1a1a1;
+  color: #666666;
+  font-size: 2rem;
+  padding-bottom: 15px;
+`;
```

Comment:
p2: padding 값이 위에도 쓰이고 밑에도 따로 설정해주셨는데 한번에 설정을 해주시면 좋을 듯 보입니다.
```suggestion
const PurchaseHistoryCategory = styled.div`
  display: flex;
  width: 100%;
  border-bottom: 1px solid #a1a1a1;
  color: #666666;
  font-size: 2rem;
  padding: 0 52px 15px 0;
`;
```